.env

```diff
@@ -1 +1 @@
-MONGOHQ_URL=mongo://localhost:27017/cfh
\ No newline at end of file
+MONGOHQ_URL=mongodb://localhost:27017/cfh
```

Comment:
This file should be deleted, .env file is specific to the application configuration on your local machine!!!